compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/MergeComponentProcessor.kt

```diff
@@ -55,8 +56,30 @@ internal class MergeComponentProcessor(
     private val codeGenerator: CodeGenerator,
     override val logger: KSPLogger,
     private val contributesSubcomponentProcessor: ContributesSubcomponentProcessor,
+    options: Map<String, String>,
 ) : SymbolProcessor, ContextAware {
 
+    private val defaultContributingAnnotations = buildSet {
```

Comment:
pulled these up opportunistically to cache them across rounds

Reply:
Good!